src/hardhat/hre/W3fHardhatPlugin.ts

```diff
@@ -97,11 +97,17 @@ export class Web3FunctionHardhat {
       log,
     };
     if (operation === "onFail") {
+      //Todo: accept arguments
```

Comment:
What is this todo for?

Reply:
I was thinking if we need to allow users to pass those onFail reasons & arguments